File: Pemrograman-1/Modul 5/PY/PRAK504-2310817210014-DamarjatiSuryoLaksono.py
def reverse (num):
    reversed = 0 
    while num !=0 :
        digit = num % 10
        reversed = reversed * 10 + digit 
        num //= 10
    return reversed 
A,B = map(int,input("").split(" "))
A = reverse(A)
B = reverse(B)
C = A + B 
print(reverse(C))

File: Pemrograman-1/Modul 5/C/PRAK504-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int reverse(int num) {
    // Lengkapi Function ini
    int reversed = 0;
    for (; num > 0; num /= 10) {
        int digit = num % 10;
        reversed = reversed * 10 + digit;
    }
    return reversed;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
    return 0;
}
